ocs_ci/ocs/cluster.py

```diff
@@ -878,3 +878,135 @@ def validate_pg_balancer():
             return False
     else:
         logging.info(f"pg_balancer is not active")
+
+
+def get_child_nodes_osd_tree(node_id, osd_tree):
+    """
+    This functions finds the children of a node from the 'ceph osd tree' and returns them as list
+
+    Args:
+        node_id (int): the id of the node for which the children to be retrieved
+        osd_tree (dict): dictionary containing the output of 'ceph osd tree'
+
+    Returns:
+        list: of 'children' of a given node_id
+    """
+    for i in range(len(osd_tree['nodes'])):
+        if osd_tree['nodes'][i]['id'] == node_id:
+            return osd_tree['nodes'][i]['children']
+
+
+def check_osds_in_hosts_osd_tree(all_hosts, osd_tree):
+    for each_host in range(len(all_hosts)):
```

Comment:
I think it will be more clear to write 'for each_host in all_hosts:' and then at line 901 replace 'all_hosts[each_host]' with 'each_host'.